lib/utilrb/pkgconfig.rb

```diff
@@ -207,16 +216,33 @@ def self.parse_dependencies(string)
                     end
                 end
             end
+            
+            if memo.empty?
+                depth = 1
+            else
+                depth = memo.values.max + 1
+            end
+
+            packages.map do |dep|
+                dep = dep.strip
+                if !memo.key?(dep)
```

Comment:
`if !` is written `unless`, and move to a one-line form for small one-line conditionals, e.g.

```
memo[dep] = depth unless memo[dep]
```

However, in this case, since values are not `nil`, the idiomatic thing to write would be

```
memo[dep] ||= depth
```